### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by enjeck** [cert-manager/website#917]
The links on the GSoD website are broken. The project proposal should link to https://cert-manager.io/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index/, not https://cert-manager.io/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/. The budget link is broken too. 

If an update can't be made from the GSoD website, I could try updating the current link to match it. Maybe adding a redirect to `next.config.js`? Not sure yet!

**Comment by maelvls** [comment]
Hi! Thanks for reporting the dead links. We performed a large migration from the "old" website design that used to rely on Hugo to the "new" website that relies on Next.js, and it seems like the links that we gave when submitting the Google Season of Docs are now broken.

In red the "expected" links, and in green the link that "works":
```diff
+https://cert-manager.io/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/
-https://cert-manager.io/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index/
```

I'll investigate and try to fix it in the next hour.
